docs/4.0/trustedclusters.md

```diff
@@ -144,6 +144,7 @@ Consider the following:
 * Every user in Teleport Enterprise is assigned a role.
 * When creating a _trusted cluster_ resource, the administrator of "east" must
   define how roles from "main" map to roles on "east".
+* To update role map for an existing _trusted cluster_ delete and re-create the _trusted cluster_ with the updated role map
```

Comment:
I would consider also adding a section after the Example, that is dedicated section on Updating trust. Between the last RBAC Trusted Cluster and `Using Trusted Clusters` 
![image](https://user-images.githubusercontent.com/559288/65189642-68994b80-da26-11e9-96ef-8f3414928f3a.png) 

Possibly with `tctl rm main-cluster.yaml` -> `tctl create main-user-updated-role.yaml` as an example.

Reply:
Got it, just made this update